test/LibraryManager.IntegrationTest/Services/InstallDialogTestService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.Test.Apex.Services;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Shell;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Web.LibraryManager.Vsix.UI;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest.Services
+{
+    /// <summary>
+    /// Test service for add client side libraries dialog.
+    /// </summary>
+    [Export(typeof(InstallDialogTestService))]
+    public class InstallDialogTestService : VisualStudioTestService
+    {
+        public InstallDialogTestExtension OpenDialog()
+        {
+            Guid guid = Guid.Parse("44ee7bda-abda-486e-a5fe-4dd3f4cefac1");
+            uint commandId = 0x0100;
+
+            Task.Factory.StartNew(() =>
```

Comment:
>Task.Factory.StartNew(() => [](start = 12, length = 27)

This either needs to be commented or moved to a new method like ExecuteCommandAsync (this method wouldn't need the resultant task, but I could see how that would be useful)

Reply:
Updated in iteration 3

---
In reply to: [205526398](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205526398) [](ancestors = 205526398)

Reply:
Would be good to add a comment that we don't wait for the completion of the command as it doesn't come back until the dialog is dismissed.

---
In reply to: [205864201](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205864201) [](ancestors = 205864201,205526398)

Reply:
Added comment in iteration 4

---
In reply to: [205905814](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205905814) [](ancestors = 205905814,205864201,205526398)